lib/webmachine/constants.rb

```diff
@@ -3,73 +3,73 @@
   CRLF = "\r\n".freeze
 
   # HTTP Content-Type
-  CONTENT_TYPE = 'Content-Type'.freeze
+  CONTENT_TYPE = "Content-Type".freeze
 
   # Default Content-Type
-  TEXT_HTML = 'text/html'.freeze
+  TEXT_HTML = "text/html".freeze
 
   # HTTP Date
-  DATE = 'Date'.freeze
+  DATE = "Date".freeze
 
   # HTTP Transfer-Encoding
-  TRANSFER_ENCODING = 'Transfer-Encoding'.freeze
+  TRANSFER_ENCODING = "Transfer-Encoding".freeze
 
   # HTTP Content-Length
-  CONTENT_LENGTH = 'Content-Length'.freeze
+  CONTENT_LENGTH = "Content-Length".freeze
 
   # A underscore
-  UNDERSCORE = '_'.freeze
+  UNDERSCORE = "_".freeze
 
   # A dash
-  DASH = '-'.freeze
+  DASH = "-".freeze
 
   # A Slash
-  SLASH = '/'.freeze
+  SLASH = "/".freeze
 
-  MATCHES_ALL = '*/*'.freeze
+  MATCHES_ALL = "*/*".freeze
 
-  GET_METHOD     = "GET"
-  HEAD_METHOD    = "HEAD"
-  POST_METHOD    = "POST"
-  PUT_METHOD     = "PUT"
-  DELETE_METHOD  = "DELETE"
+  GET_METHOD = "GET"
```

Comment:
This might not be improvement. I can disable the indentation rules if that's preferred.